platform/admin/core/admin-core-insecuredefaults/pom.xml

```diff
@@ -22,39 +22,8 @@
     <artifactId>admin-core-insecuredefaults</artifactId>
     <name>DDF :: Admin :: Core :: Insecure Defaults</name>
     <description>Verifies insecure default configurations have been changed.</description>
-    <packaging>bundle</packaging>
```

Comment:
Because the `InsecureDefaultsService` is now instantiated in the `AlertServiceBean` this module no longer needs to be a bundle. It is now embedded in `admin-core-alert-service`.